### ISSUE TITLE
Maps do not show light as bright as they should, only about 5% brightness.

### ISSUE THREAD

**Issue by Ramses-II** [dev7355608/perfect-vision#213]
Hey, 
after the last update (I skipped a few versions) pre-placed light sources (that were imported in a universal-vtt file) do not have the correct brightness. They appear extremely dark, at about 5% of their supposed brightness.
As I just updated, everything is on latest stable. Foundry and this module. (I can provide build codes, just don't have them at hand right now)
Process:
Export map from Dungeondraft, "export light" turned off or on, no influence on the issue. 
Imported via Battlemap-Importer
Light sources are too dim
Placing new light has the correct behaviour
Copying ALL properties from the preplaced lights to the manually added light do not replicate the problem IIRC (did the tests a few hours ago)

I checked old maps and the problem is the same, so any of my maps that were imported before the updates behave wrong. Only manual light sources, either placed or from tokens, work correct. 

Disabling this module solves the issues, reenabling it brings the problems back. Reproducible. 

TL:DR. Lights from univeral-vtt sources do not shine bright, manually placed ones do. 

If you need any more info, I can try to provide it. 
Greetings Ramses

**Comment by dev7355608** [comment]
Could you share one of those scenes with me?

**Comment by Ramses-II** [comment]
Hey,

I have uploaded a zip with the .dd2vtt, three pictures and the json from the scene. I hope it helps. 
If you need anything else, please give me a shout!
https://drive.google.com/drive/folders/18AkXYbwwL5250uVnxHVBDhd1OCzh2JwX?usp=sharing
Greetings Ramses

**Comment by dev7355608** [comment]
I seem to have awakened a dormant bug in core Foundry with a recent PV release. The color intensity of the light source was affecting the brightness of the light. All your lights in your scene where slightly colored yellow with color intensity set to 0.05, which also made the lights really dark. I corrected this in the latest release: color intensity has no longer effect on the illumination brightness of the light.

Thanks for the report!

**Comment by Ramses-II** [comment]
Thanks a lot for the very fast fix!
